pihole

```diff
@@ -185,6 +185,19 @@ piholeLogging() {
 }
 
 piholeStatus() {
+  if [[ $(netstat -plnt | grep -c ':53') > 0 ]]; then
```

Comment:
by adding a space after `:53` we make sure grep will exclude ports starting with `53`. On my system :

```
$ netstat -plnu | grep -c ':53'
udp        0      0 0.0.0.0:5317            0.0.0.0:*                           -
udp        0      0 0.0.0.0:5353            0.0.0.0:*                           -
udp6       0      0 :::53                   :::*                                -
udp6       0      0 :::5353                 :::*                                -
```
while 

```
$ netstat -plnu | grep ':53 '
udp6       0      0 :::53                   :::*                                -
```